File: Apps/MVC/App/views/user/Panel.js
Ext.define('Mvc.views.user.Panel', {
	extend: 'Dextop.Panel',

	title: 'Users',
	border: false,
	alias: 'widget.userspanel',
	closable: true,

	initComponent: function () {
		var grid = Ext.create('Dextop.ux.SwissArmyGrid', {
			remote: this.remote,
			paging: true,
			border: false,
			editing: 'row',
			tbar: ['add', 'edit', 'remove'],
			storeOptions: {
				pageSize: 10,
				autoLoad: true,
				autoSync: true
			},
			editingOptions: {
				clicksToEdit: 1
			}
		});

		Ext.apply(this, {
			layout: 'fit',
			items: grid
		});

		this.callParent(arguments);
	}
});

File: Apps/MVC/App/views/user/Panel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Codaxy.Dextop;
using Codaxy.Dextop.Remoting;
using Codaxy.Dextop.Data;
using Mvc.App.model;

namespace Mvc.App.views.user
{
    public class Panel : DextopPanel
    {
        [DextopRemotableConstructor(alias = "users")]
        public Panel()
        {
            
        }

        public override void InitRemotable(DextopRemote remote, DextopConfig config)
        {
            base.InitRemotable(remote, config);
            Remote.AddStore("model", new Codaxy.Dextop.Data.DextopMemoryDataProxy<User, int>(a=>a.Id, (id) => { return ++id; }));            
        }
    }
}